kubernetes/linux/Dockerfile.multiarch

```diff
@@ -43,4 +43,10 @@ RUN trivy rootfs --exit-code 1 --ignore-unfixed --no-progress --severity HIGH,CR
 

```

Comment:
This wont catch the OS/system CVEs - this will catch only lib & app CVEs. Will your change apply to this PR as well , so we can see what this PR is catching with your changes ?

Reply:
i thought using rootfs scans root filesystem thereby scanning OS/system CVEs? (https://aquasecurity.github.io/trivy/v0.22.0/vulnerability/scanning/rootfs/)

Yes, this change will apply to this PR itself. The ContainerInsights-MultiArch-Dev check will fail

Reply:
can we take a previous image and scan both ways to see the results ? I am sure they will be different (both ways).

Reply:
1. Results scanning docker image mcr.microsoft.com/azuremonitor/containerinsights/ciprod:ciprod03172022

![image](https://user-images.githubusercontent.com/10297394/164538114-ed876b41-9dec-4a81-8a13-896758e97277.png)

2. Results scanning filesystem using rootfs

![image](https://user-images.githubusercontent.com/10297394/164538416-4834d2bf-2f9e-4122-abc4-68a5f82b21f3.png)

Reply:
Thanks. That looks better. We should scan for MEDIUM as well. As customers report on Medium as well (though it will be best effort on our side). If you can change in ADO checks, i will change in other places to ionclude medium.

Reply:
we do --exit-code 1 with trivy to fail the build in case we find any vulnerabilities. If we include medium as well the builds will not pass for medium vulnerabilities. is that what we want?

Reply:
@vishiy can you take a look now? I have added the new checks and a trivyignore file. Currently it contains CVEs being reported in /opt/telegraf binary (except the newer go crypto module one)